frontend/src/layout/Sidebar.js

```diff
@@ -193,6 +194,11 @@ export function Sidebar({ user, sidebarCollapsed, setSidebarCollapsed }) {
                             <span className="sidebar-label">{'Cohorts'}</span>
                             <Link to={'/people/cohorts'} onClick={collapseSidebar} />
                         </Menu.Item>
+                        <Menu.Item key="retention" style={itemStyle} data-attr="menu-item-retention">
```

Comment:
You might have to add this to `submenuOverride` above, as the "people" menu doesn't stay open at the moment

Reply:
Great catch, thanks. :)